runelite-client/src/main/java/net/runelite/client/plugins/playerindicators/PlayerIndicatorsOverlay.java

```diff
@@ -76,7 +76,7 @@ public Dimension render(Graphics2D graphics, Point parent)
 					renderPlayerOverlay(graphics, player, config.getOwnNameColor());
 				}
 			}
-			else if (config.drawFriendNames() && player.isFriend())
+			else if (config.drawFriendNames() && client.isFriended(player.getName(), false))
```

Comment:
Hello i have no idea how to comment on here, but i would like to  kindly ask you how do i make these changes?? I want runelite to show offline friends on minimap badly and i'm going crazy trying to figure out how to do it...

Reply:
This is a feature rejected by the team. We will not be assisting you with making such changes.